db/migrations/00015_add_role_to_member_task.go

```diff
@@ -26,6 +26,8 @@ func Up00015(tx *sql.Tx) error {
 
 	gormdb.AutoMigrate(&sprintMemberTask{})
 
+	gormdb.Model(&models.Task{}).AddIndex("idx_tasks_done_at", "done_at")
```

Comment:
why was this added here, should not it be in the new migration added below

Reply:
I accidentally added this here, nice catch!. Fixed